presto-main/src/main/java/com/facebook/presto/operator/scalar/ReflectionParametricScalar.java

```diff
@@ -100,44 +99,74 @@
     private final List<Implementation> specializedImplementations;
     // These are generic implementations
     private final List<Implementation> implementations;
-    private final String description;
-    private final boolean hidden;
-    private final boolean deterministic;
+
+    public static class ScalarHeaderAndImplementations
+    {
+        private final ScalarImplementationHeader header;
+        private final List<Method> methods;
+
+        public ScalarHeaderAndImplementations(ScalarImplementationHeader header, List<Method> methods) {
```

Comment:
`{` should be on next line. Can you run the Maven build for each commit? I think there's a style checker for this